sdk/core/core-lro/test/lro.spec.ts

```diff
@@ -1037,9 +1037,9 @@ matrix(
               assert.equal(result.id, "100");
             });
 
-            it("should handle patchAsync", async () => {
-              const resourceLocationPath = `/patchasync/succeeded`;
-              const pollingPath = `/patchasync/operationresults/123`;
+            it("should handle patchAsyncLocationHeader", async () => {
```

Comment:
This is the standard PATCH LRO, https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#updating-using-patch.